samples/Azure.ResourceManager.Sample/Generated/SshPublicKeyResourceContainer.cs

```diff
@@ -7,11 +7,11 @@
 
 namespace Azure.ResourceManager.Sample
 {
-    /// <summary> A class representing collection of SshPublicKeys and their operations over a [ParentResource]. </summary>
-    public partial class SshPublicKeysContainer
+    /// <summary> A class representing collection of SshPublicKeyResource and their operations over a [ParentResource]. </summary>
+    public partial class SshPublicKeyResourceContainer
```

Comment:
Why did this change to SshPublicKeyResourceContainer instead of SshPublicKeyContainer

Reply:
https://github.com/AME-Redmond/autorest.csharp/blob/mgmt-track2/samples/Azure.ResourceManager.Sample/sample.json#L1248